documentation/asciidoc/computers/configuration/headless.adoc

```diff
@@ -1,77 +1,40 @@
+=== Remote access
 
-WARNING: After your Raspberry Pi is connected to power, make sure to wait a few (up to 5) minutes for it to boot up and register on the network.
+With no keyboard or monitor, you need a way to xref:remote-access.adoc[remotely control] your headless Raspberry Pi. On first boot, the only option is SSH. To enable SSH on a fresh installation of Raspberry Pi OS, choose one of the following methods:
 
-A xref:configuration.adoc#wireless-networking-command-line[`wpa_supplicant.conf`] file example:
+* enable SSH in the Advanced Menu of Raspberry Pi Imager, then enter a username and password
+* create a file named `ssh` at the root of the SD card, then configure a user manually with `userconf.txt` following the instructions in the section below
 
-----
-ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
-country=<Insert 2 letter ISO 3166-1 country code here>
-update_config=1
-
-network={
- ssid="<Name of your wireless LAN>"
- psk="<Password for your wireless LAN>"
-}
-----
-
-Where the country code should be set the two letter ISO/IEC alpha2 code for the country in which you are using, e.g.
-
-* GB (United Kingdom)
-* FR (France)
-* DE (Germany)
-* US (United States)
-* SE (Sweden)
-
-Here is a more elaborate example that should work for most typical wpa2 personal networks. This template below works for 2.4ghz/5ghz hidden or not networks. The utilization of quotes around the ssid - psk can help avoid any oddities if your network ssid or password has special chars (! @ # $ etc)
-
-----
-ctrl_interface=DIR=/var/run/wpa_supplicant GROUP=netdev
-update_config=1
-country=<Insert 2 letter ISO 3166-1 country code here>
-
-network={
-        scan_ssid=1
-        ssid="<Name of your wireless LAN>"
-        psk="<Password for your wireless LAN>"
-        proto=RSN
-        key_mgmt=WPA-PSK
-        pairwise=CCMP
-        auth_alg=OPEN
-}
-----
-
-NOTE: Some older Raspberry Pi boards and some USB wireless dongles do not support 5GHz networks.
-
-NOTE: With no keyboard or monitor, you will need some way of xref:remote-access.adoc[remotely accessing] your headless Raspberry Pi. For headless setup, SSH can be enabled by placing a file named `ssh`, without any extension, onto the boot folder of the SD Card. For more information see the section on xref:remote-access.adoc#ssh[setting up an SSH server].
+For more information, see xref:remote-access.adoc#ssh[set up an SSH server]. Once you've connected over SSH, you can use `raspi-config` to xref:remote-access.adoc#vnc[enable VNC] if you'd prefer a graphical desktop environment.
 
 [[configuring-a-user]]
-=== Configuring a User
+=== Configure a user manually
```

Comment:
Is it worth simply folding this entire section into the second bullet-point in the "Remote access" section, rather than making this a whole separate section in its own right?

Reply:
hot take: yes! It's too much scope creep for the above section, let alone a bullet point. Breaks up the flow of the above section too much.

I _would_ be willing to add an extra `=` to drop it one header level and nest it within that other section, though. But I don't think it's necessary since it is technically a discrete task.

Reply:
Should the configuring a user thing go at the *top of the whole thing* because if you don't configure a user your headless installation is useless.

So "Setting up a headless Raspberry Pi"

First you should configure your user account...

Reply:
So just between the lede, and the SSH section. You can't SSH without a user, so the user stuff should go before the SSH stuff?

Reply:
I was trying to suggest that we make the `userconf.txt` instructions less prominent, since most people will just be doing this via Raspberry Pi Imager.

Reply:
User setup in general? Yes. Manual user setup with userconf? Especially with the latest Bookworm changes for wireless network setup on initial boot, decreasingly important and worthy of pushing down the page.

Reply:
Yes, that's true if we have to point users at Imager to configure the wifi anyway, they should also configure the user there at the same time (and SSH!). Then we can demphasis both the `userconf.txt` and `ssh` files.